### DISCUSSION TITLE
Feature: Co-Tags, how should it work?

### DISCUSSION THREAD

**Discussion by SkepticMystic** [SkepticMystic/graph-analysis#22] [Ideas]
We have received a few feature requests about including tags in the analysis, but aren't quite sure how to implement it. There are a couple of proposals as it stands:

## Integrate coTags into Co-Citations

This would gives extra weighting to notes with the same tags
![](https://media.discordapp.net/attachments/809479003835596851/905772793214410792/unknown.png)

Discussion:

- Tags could be given a customisable weighting (0 - 1)

## Create a Global coTags View

This would list out every tag in your vault which, when clicked, would open a dropdown showing which other tags appear in the same notes. Like this, but for every note:

![](https://media.discordapp.net/attachments/809479003835596851/905748843910561792/unknown.png?width=720&height=245)

Discussion:

- How would you handle nested tags?

---

Please share your thoughts on these two proposals, and do suggest alternatives that you think of!

**Comment by licht1stein** [comment]
I think if notes A and B have the same tag #sample, it should be equal to both A and B linking to note SAMPLE.

A -> SAMPLE <- B
A -> #sample <- B

**Comment by thoresson** [comment]
I'm one of the them who has requested support for tags as well. My use case relates to how I log my workdays in my daily notes. I've settled on a syntax where I use tags do categorize the kind of work and pages for persons, projects etc. Typical entries in the log would look something like this:

- 10:00am – #Meeting with [[Person 1]] about [[Project X]]
- 10:30am – #Research for [[Project X]]
- 11:15am – #Writing on the project plan for [[Project X]]
- 1:45pm – #Meeting with [[Person 1]] and [[Person 2]] about [[Project Y]]

I also have some tagged headings in my daily notes template. One example is

**Questions in need of an answer #questions**
For [[Project X]], I have to find a way do to X.

What I've been hoping for when starting working with this syntax is that there eventually would be a way to filter out all meetings I've had with a certain person and/or about a certain project and also filtering out questions regarding a person/project.

When I found this plugin it seems like the closest so far, and hence my questions about support for tags. :)

**Reply by HEmile** [reply]
Ah, that's very interesting! This is certainly a use case we had not considered before, yet i think it's a natural fit. 

In your example, if you'd go to the note of Person 1, you'd then see a #Meeting entry in the co-citation entry, which can be expanded to find all lines where you talk about #Meetings with [[Person 1]]. We will likely implement that (optional, but default).

**Reply by thoresson** [reply]
Excellent! :)

**Comment by SkepticMystic** [comment]
@licht1stein @thoresson checkout 0.6.4, co-citations now show coTags!
